pvanalysis/_pvanalysis.py

```diff
@@ -1214,7 +1203,9 @@ def plot_model(self, ax=None, loglog: bool = False,
             print('Please input ax.')
             return -1
         if model is None: model = self.model
-        if popt  == []: popt = self.popt[method][0]
+        if popt  == []:
+            popt = self.popt[method][0].copy()
+            popt[4] -= self.avevsys
```

Comment:
If self.vsys_opt is defined in any case, popt[4] -= self.avevsys --> popt[4] = self.vsys_opt?